lisa/sut_orchestrator/libvirt/transformers.py

```diff
@@ -505,6 +528,14 @@ def _install_libvirt(runbook: schema.TypedSchema, node: Node, log: Logger) -> No
             node.execute("systemctl disable apparmor", shell=True, sudo=True)
         node.execute("systemctl enable libvirtd", shell=True, sudo=True)
         node.reboot()
+        if isinstance(node.os, CBLMariner):
+            # After reboot, libvirtd service is in failed state and needs to
+            # be restarted manually. Doing it immediately after restart
+            # fails. So wait for a while before restarting libvirtd.
+            # This is an issue in Mariner and below lines can be removed once
+            # it has been addressed.
+            time.sleep(10)
```

Comment:
We generally do not hardcore sleep but instead follow something like: sleep for 1 sec, check reboot status, .... You can look at the Azure reboot function call. Can you please look if it is possible to implement something similar here?

Reply:
Done.